config/environments/development.rb

```diff
@@ -74,4 +74,6 @@
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
   config.force_ssl = false
+
+  config.secret_share_enabled = true
```

Comment:
Maybe it should be disabled by default on `master` (so it doesn't disrupt or confuse other contributors) and only manually changed to `true` locally by people who are working on the feature.

You could comment out this line and add a comment saying to uncomment it if you are working on Secret Share Moments.

Reply:
^ I like this idea.